index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&display=swap"
+        rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+    <header>
+        <nav>
+            <p class="john">John Doe</p>
+            <a href="#" class="menu-icon">
+                <svg class="menu-icon" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg"
+                    xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="20px" height="20px"
+                    viewBox="0 0 122.879 103.609" enable-background="new 0 0 122.879 103.609" xml:space="preserve">
+                    <g>
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                            d="M10.368,0h102.144c5.703,0,10.367,4.665,10.367,10.367v0 c0,5.702-4.664,10.368-10.367,10.368H10.368C4.666,20.735,0,16.07,0,10.368v0C0,4.665,4.666,0,10.368,0L10.368,0z M10.368,82.875 h102.144c5.703,0,10.367,4.665,10.367,10.367l0,0c0,5.702-4.664,10.367-10.367,10.367H10.368C4.666,103.609,0,98.944,0,93.242l0,0 C0,87.54,4.666,82.875,10.368,82.875L10.368,82.875z M10.368,41.438h102.144c5.703,0,10.367,4.665,10.367,10.367l0,0 c0,5.702-4.664,10.368-10.367,10.368H10.368C4.666,62.173,0,57.507,0,51.805l0,0C0,46.103,4.666,41.438,10.368,41.438 L10.368,41.438z" />
+                    </g>
+                </svg>
+            </a>
+        </nav>
+    </header>
+    <main>
+        <h1>I’m Elmar Abdul</h1>

```

Comment:
- [ ] **The main container of headline section should be a `section` tag**

This will have various benefits. Firstly, it will group all the content inside of headline element into one main container. Making it easier to manage the overall layout. Secondly, the `section` tag makes your HTML more semantic. Having semantic HTML results in better search engine optimization. It will also ensure better accessibility across different devices and platforms. You can refer to [this article](https://www.w3schools.com/html/html5_semantic_elements.asp) to understand more about semantic HTML.